speedcord/ext/typing/context.pyi

```diff
@@ -35,8 +35,7 @@ class MessageContext:
     application: Optional[Dict[str, Any]]
     message_reference: Optional[Dict[str, int]]
     flags: Optional[int]
-    ...
 
-    async def send(self, *, content: str = None, nonce: Union[int, str] = None, tts: bool = None,
-                   embed: Dict[str, Any] = None, allowed_mentions: Dict[str, Any] = None):
+    async def send(self, *, content: str = ..., nonce: Union[int, str] = ..., tts: bool = ...,
```

Comment:
Is there a specific reason you want to use `...` instead of None here?

Reply:
It's in PEP 484 that you can [use ellipsis as a default value](https://www.python.org/dev/peps/pep-0484/#arbitrary-argument-lists-and-default-argument-values): 

Additionally, `mypy` and `pyright` revealed using `None` as errors.

Reply:
Oh okay, my bad. Best to keep it to `...` then